composer.json

```diff
@@ -6,7 +6,6 @@
         "php": ">=7.0"
     },
     "require-dev": {
-        "ext-xdebug": "*",
```

Comment:
@Lewiscowles1986, please consider this explanation.

If we define this `xdebug` extension, it will let `Composer` check whether the extension is loaded correctly.

And it's not necessary to do this checking when doing `unit` tests work.

I remove this line finally.